lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -95,14 +114,14 @@ def self.create_release(repository, version, release_notes, assets, prerelease)
       # @return [String] The path of the downloaded file, or nil if something went wrong
       #
       def self.download_file_from_tag(repository:, tag:, file_path:, download_folder:)
-        repository = repository.delete_prefix('/').chomp('/').concat('/')
-        file_path = file_path.delete_prefix('/').chomp('/').concat('/')
-        tag = tag.concat('/')
+        repository = repository.delete_prefix('/').chomp('/')
+        file_path = file_path.delete_prefix('/').chomp('/')
         file_name = File.basename(file_path)
         download_path = File.join(download_folder, file_name)
 
         begin
-          open(URI.join('https://raw.githubusercontent.com/', repository, tag, file_path).to_s.chomp('/')) do |remote_file|
+          uri = URI.parse("https://raw.githubusercontent.com/#{repository}/#{tag}/#{file_path}")
+          uri.open do |remote_file|
```

Comment:
These changes are unrelated to this PR but were required to fix a severe rubocop violation otherwise reported as ` Security/Open: The use of Kernel#open is a serious security risk.` – see [more info here](https://docs.rubocop.org/rubocop/cops_security.html#securityopen).